ocs_ci/ocs/registry.py

```diff
@@ -279,3 +291,22 @@ def image_rm(registry_path):
     ocp_obj = ocp.OCP()
     ocp_obj.exec_oc_debug_cmd(node=master_list[0], cmd_list=cmd_list)
     logger.info(f"Image {registry_path} rm successful")
+
+
+def check_image_in_registry(image_url):
+    """
+    Function to check either image present in registry or not
+
+    Args:
+        image_url (str): Image url to be verified
+
+    Returns:
+        True : Returns True if present
```

Comment:
if this function returns a bool, it should return False under a certain condition

Reply:
@ebenahar will remove the return True from the code. anyway there is check for image_url_present else there will be assert.